packages/flutterfire_ui/lib/src/i10n/lang/de.dart

```diff
@@ -242,6 +252,15 @@ class DeLocalizations extends FlutterFireUILocalizationLabels {
     this.signInWithEmailLinkSentText =
         'Wir haben Ihnen eine E-Mail mit einem Magic Link geschickt. Prüfen Sie Ihre E-Mail und folgen Sie dem Link, um sich anzumelden',
     this.sendLinkButtonLabel = 'Magic Link senden',
+    this.credentialAlreadyInUseErrorText =
+        'This credential is already associated with a different user account.',
+    this.verifyEmailTitleText = 'Verify your email',
+    this.emailVerificationInstructionText =
+        'A verification email has been sent to your email address. '
+            'Please check your email and click on the link to verify '
+            'your email address.',
+    this.couldntVerifyEmailText = "We couldn't verify your email address. ",
+    this.resendVerificationEmailButtonLabel = 'Resend verification email',
```

Comment:
this.credentialAlreadyInUseErrorText =
        'Diese Anmeldedaten werden bereits für ein anderes Konto verwendet.',
    this.verifyEmailTitleText = 'Bestätigen Sie Ihre E-Mail',
    this.emailVerificationInstructionText =
        'Es wurde eine Bestätigungs-E-Mail an Ihre E-Mail-Adresse gesendet. Bitte prüfen Sie Ihre E-Mail und klicken Sie auf den Link, um Ihre E-Mail-Adresse zu bestätigen.',
    this.couldntVerifyEmailText =
        'Wir konnten Ihre E-Mail-Adresse nicht verifizieren.',
    this.resendVerificationEmailButtonLabel =
        'Bestätigungs-E-Mail erneut senden',